lando_util/organize_project/organizer.py

```diff
@@ -42,7 +43,9 @@ def __init__(self, cmdfile):
         data = json.load(cmdfile)
         self.bespin_job_id = data['bespin_job_id']  # bespin job id of the job this output project is for
         self.destination_dir = data['destination_dir']  # directory where we will add files/folders
-        self.workflow_path = data['workflow_path']  # path to the workflow we ran
+        self.workflow_path = data['workflow_path']  # path to downloaded workflow either a zip file or packed cwl
```

Comment:
It looks like `workflow_path` here is a local path to an already downloaded file, can you confirm that's the case? I ask because we have `workflow_path` in bespin-api and lando, where it describes the path to the workflow object in the archive/packed workflow. I didn't realize `workflow_path` was already in use here, and would have probably picked a different name to avoid the confusion.

Reply:
I renamed `workflow_path` to `downloaded_workflow_path` which better matches the comment.